### DISCUSSION TITLE
How we can enable h2-console using quarkus?

### DISCUSSION THREAD

**Discussion by ranjeet-kush** [quarkusio/quarkus#40181] [Q&A]
Currently I am using Quarkus 3.2 where I have tried enabling the h2 console by adding properties for db-kind = h2 in application.properties file but couldn't open h2-console UI. Please suggest a way to perform this operation where we can check curd operations using H2 db.

**Comment by josealonso** [comment]
The same problem here.

**Reply by gengxiaohong** [reply]
same issue I faced
any solution to resolve this?

**Comment by gengxiaohong** [comment]
same issue I faced
any solution to resolve this?
